### DISCUSSION TITLE
FCSys on Openmodelica

### DISCUSSION THREAD

**Discussion by Yahiaassam** [Virtual-FCS/VirtualFCS#33] [Q&A] [answered]
Hi, i'am yahia, i'am working  on openmodelica and i need the virtual FCS, so i would like to know in which version this library work in openmodelica ? actually i use the 1.18 version;
Thanks for your needs 
have a good day.

**Comment by kvid** [comment, marked as answer]
Hi, again. It seems you asked the same question here 3 days ago, but I discovered and answered your #34 issue today first, so I just repeat my answer here:
> OM 1.14 should work. See more details in #1.

**Reply by kvid** [reply]
Update: [VirtualFCS v2.0.0](https://github.com/Virtual-FCS/VirtualFCS/releases/tag/v2.0.0), supporting OM v1.20 for Windows, has now been released.
